samples/src/source/screens/TaskScreenVM.bs

```diff
@@ -0,0 +1,48 @@
+import "pkg:/source/roku_modules/maestro/mx/BaseViewModel.brs"
+import "pkg:/source/roku_modules/maestro/mx/VMTransformFunctions.brs"
+'
+namespace screens
+  class TaskScreenVM extends mx.BaseViewModel
+
+    public focusedIndex = -1
+    public items        = createObject("roSGNode", "ContentNode")
+    public hasItems     = false
+    public loadDataTask
```

Comment:
bare in mind that the correct way to declare this is with types. maestro has type polyfills (see Types-Polyfill.d.brs)
`public loadDataTas ac mc.types.node`